### ISSUE TITLE
[Bug] `yarn config -v` is advised if .yarnrc contains an invalid option, but it cannot run

### ISSUE THREAD

**Issue by StephanBijzitter** [yarnpkg/berry#117]
**Describe the bug**

When adding an unknown option to `.yarnrc`, yarn complains about the option and tells the user to run `yarn config -v` to see which options are available.

However, running this command results in the same error. Usually I'm quite in love with fail-fast behaviours, but in this case an exception _might_ be nice.

**To Reproduce**
* Add any unknown option to .yarnrc
* Run `yarn config -v`

**Screenshots**

```
Stephans-MacBook-Pro:redacted stephan$ yarn
Error: Unrecognized configuration settings found: npmAlwaysAuth - run "yarn config -v" to see the list of settings supported in Yarn (in /Users/stephan/redacted/.yarnrc)

Stephans-MacBook-Pro:Pro:redacted stephan$ yarn config -v
Error: Unrecognized configuration settings found: npmAlwaysAuth - run "yarn config -v" to see the list of settings supported in Yarn (in /Users/stephan/redacted/.yarnrc)
```

**Environment if relevant (please complete the following information):**

 - OS: OSX
 - Node version 12.1.0
 - Yarn version `yarn --version` shows v2.0.0; I downloaded this using `yarn policies set-version berry`

**Comment by arcanis** [comment]
Good point - we probably should add a `strict` option to `Configuration.find` (defaults to `true`) that would allow for invalid keys (I think they should be buffered into an `invalid` property of the configuration object so that `yarn config` could print a list of them all).

**Comment by arcanis** [comment]
Fixed in #118 👍
